modules/cluster-logging-rn-5.6.5.adoc

```diff
@@ -0,0 +1,27 @@
+//module included in cluster-logging-release-notes.adoc
+:content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-5{context}"]
+= Logging 5.6.5
+This release includes link:https://access.redhat.com/errata/RHSA-2023:1953[OpenShift Logging Bug Fix Release 5.6.5].
+
+[id="openshift-logging-5-6-5-bug-fixes"]
+== Bug fixes
+* Before this update, the template definitions prevented Elasticsearch from indexing some labels and namespace_labels, causing issues with data ingestion. With this update, the fix replaces dots and slashes in labels to ensure proper ingestion, effectively resolving the issue. (link:https://issues.redhat.com/browse/LOG-3419[LOG-3419])
+
+* Before this update, if the Logs page of the OpenShift Web Console failed to connect to the LokiStack, a generic error message was displayed, providing no additional context or troubleshooting suggestions. With this update, the error message has been enhanced to include more specific details and recommendations for troubleshooting. (link:https://issues.redhat.com/browse/LOG-3750[LOG-3750])
+
+* Before this update, the selection of an invalid time range format was allowed, preventing the user from selecting a valid range. With this update, time formats are now validated, enabling users to select a valid range. If an invalid time range format is selected, an error message is displayed to the user. (link:https://issues.redhat.com/browse/LOG-3583[LOG-3583])
```

Comment:
I am not sure how a lack of validation "prevents" the user from selecting a valid range. Maybe "making it harder for the user to select a valid range"?